ide/lsp.client/src/org/netbeans/modules/lsp/client/LSPBindings.java

```diff
@@ -108,7 +110,7 @@ public class LSPBindings {
     private static final int LSP_KEEP_ALIVE_MINUTES = 10;
     private static final int INVALID_START_TIME = 1 * 60 * 1000;
     private static final int INVALID_START_MAX_COUNT = 5;
-    private static final RequestProcessor WORKER = new RequestProcessor(LanguageClientImpl.class.getName(), 1, false, false);
+    private static final RequestProcessor IDLE_BINDINGS_WORKER = new RequestProcessor(LanguageClientImpl.class.getName() + "IDLE", 1, false, false); // NOI18N
```

Comment:
```suggestion
    private static final RequestProcessor IDLE_BINDINGS_WORKER = new RequestProcessor(LanguageClientImpl.class.getName() + ".IDLE", 1, false, false); // NOI18N
```

Nitpick: Have not looked at the names, but getName() returns a dotted name and the last part should IMHO also be separated.